Documentation/contributing/development/reviewers_committers/review_vendor.rst

```diff
@@ -0,0 +1,117 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _review_vendor:
+
+****************************
+Reviewing for @cilium/vendor
+****************************
+
+What is @cilium/vendor?
+=======================
+
+Team `@cilium/vendor <vendor_team_>`_ is a GitHub team of Cilium contributors
```

Comment:
Should we link the Slack channel somewhere?

Reply:
Good idea, done!